deploy/database/data.tourn.sql

```diff
@@ -0,0 +1,9 @@
+DELETE FROM tournament;
+
+DELETE FROM tournament_status;
+INSERT INTO tournament_status (name) VALUES
+('OPEN'),
+('ACTIVE'),
+('COMPLETE'),
+('CANCELLED'),
+('BROKEN');
```

Comment:
Adding a new database file type --- we're going to have to be careful that this is actually included in every place the others are.